airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""
+Module exposing the tests command to test airbyte-ci projects.
+"""
+
+import asyncclick as click
+from pipelines.cli.lazy_group import LazyGroup
+
+
+@click.group(
+    cls=LazyGroup,
+    help="Commands related to formatting.",
+    lazy_subcommands={
+        "java": "pipelines.airbyte_ci.format.java.commands.java",
+        "js": "pipelines.airbyte_ci.format.js.commands.js",
+        "license": "pipelines.airbyte_ci.format.license.commands.license",
+        "python": "pipelines.airbyte_ci.format.python.commands.python",
+    },
+    invoke_without_command=True,
+)
+@click.option("--fix/--check", type=bool, default=None, help="Whether to automatically fix any formatting issues detected.  [required]")
```

Comment:
Conceptually `fix` and `check` should be subcommands and not options.
Can we make `format` and command group and `fix` and `check` subcommands?
If you'd like `fix` and `check` to call language specific subcommand I think you can leverage [Click's Multi Command Pipelines](https://click.palletsprojects.com/en/8.1.x/commands/#multi-command-pipelines)

Reply:
Nice, I hadn't considered this. So something like `airbyte-ci format check java`, or `airbyte-ci format fix (all)`? 

This would also fix the weird behavior around the required `--fix` argument.

Reply:
Yep! 
I'll let you decide what's looks like the best between:
A. `check` being a subcommand
`airbyte-ci format check --java --python` (or all if no flag)`

B. `check` being a command group calling language-specific subcommands via [multi command chaining](https://click.palletsprojects.com/en/8.1.x/commands/#multi-command-chaining)
```
# When it's called it calls all language specific subcommands
airbyte-ci format check
airbyte-ci format check java
airbyte-ci format check python
```

I think I've a light preference for B

Reply:
great, thanks! went with B, though i'm still deciding between `airbyte-ci format check` and `airbyte-ci format check all`. the former looks nice but makes the discoverability of subcommands notably worse in my opinion

Reply:
Also the chaining works great!

Reply:
I like `airbyte-ci format check` , but I don't have a strong opinion